src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
```

Comment:
it seems to me that this piece of code can be **const men = people.filter((p) =>{
  if(!century) return p.sex === 'm'
  return p.sex === 'm' && Math.ceil(p.died / 100) === century
})**